src/sdk/namespace_fs.js

```diff
@@ -682,7 +682,50 @@ class NamespaceFS {
         const fs_account_config = this.set_cur_fs_account_config(object_sdk);
         await this._load_bucket(params, fs_account_config);
 
+        const can_use_temp_file = Boolean(nb_native().fs.gpfs);
```

Comment:
We want to allow mixing backends on the same system - see [backend enum](https://github.com/noobaa/noobaa-core/blob/0a1c7746ad68621fa47b53fb53ecf6b7f9016442/src/api/common_api.js#L941-L944)
Here's a suggestion with a comment to leave some reasons behind:

```js
// We define if a backend supports the tempfile+linkat flow,
// as every bucket can use a different fs backend type, we need to check 
// the type supports it, and for GPFS we check that the native library was loaded.
// We should update this as we add more backends that support this flow.
const can_use_temp_file = Boolean(fs_account_config.backend === 'GPFS' && nb_native().fs.gpfs);
```

Reply:
I don't know how to set fs_account_config.backend === 'GPFS' in my build, so I can not test it. I have a stand alone binary, and never got config to work.

Reply:
It's in src/core/nsfs.js - see below.
You can get if from a flag (no need to define it, just add to usage to be nice).


https://github.com/noobaa/noobaa-core/blob/0a1c7746ad68621fa47b53fb53ecf6b7f9016442/src/core/nsfs.js#L81-L86

so you can write:
```js
  backend: argv.backend || '',
```

Reply:
fixed

Reply:
can you add this change of nsfs.js to the PR?

Reply:
so just make it backend: 'GPFS',  ?